doxyfile_options

```diff
@@ -841,10 +841,15 @@ EXCLUDE_PATTERNS       = */tools/* \
                          */BUILD/* \
                          */rtos/TARGET_CORTEX/rtx* \
                          */cmsis/* \
-                         */FEATURE_* \
                          */features/mbedtls/* \
                          */features/storage/* \
                          */features/unsupported/* \
+                         */features/FEATURE_COMMON_PAL/* \
+                         */features/FEATURE_LWIP/* \
+                         */features/FEATURE_UVISOR/* \
+                         */FEATURE_NANOSTACK/* \
```

Comment:
There is no FEATURE_NANOSTACK, should be features/nanostack. Should we be excluding the following as well?

                         */features/filesystem/* \
                         */features/frameworks/* \
                         */features/netsocket/* \
                         */features/TESTS/* \

Reply:
@pan- Will you be adding the above as well?